docs/data-sources/connectors.md

```diff
@@ -22,6 +29,12 @@ data "twingate_connectors" "all" {}
 ### Optional
 
 - `connectors` (Attributes List) List of Connectors (see [below for nested schema](#nestedatt--connectors))
```

Comment:
The `connectors` attribute is not an input. This type needs to be updated so it will be moved to the "Read-Only" section in the docs

Reply:
fixed